web/src/lib/i18n/en.json

```diff
@@ -816,6 +816,7 @@
   "look": "Look",
   "loop_videos": "Loop videos",
   "loop_videos_description": "Enable to automatically loop a video in the detail viewer.",
+  "main_branch_warning": "⚠️You're running a build from the main branch. We strongly recommend using a release version!",
```

Comment:
we should probably use an icon in the code and avoid embedding an emoji in the translated text imo.

Reply:
I thought so too but it'd break in RTL languages I think

Reply:
That can be fixed other ways though. There is something very brittle about emojis in text. I prefer icons over emojis in general, and this way if we wanted to change it we only do it in one place.

Reply:
Do we have a way to handle RTL for that yet? Or is it something I can leave for later? I think I prefer the emoji over an icon, but I do agree that having it in the i18n strings isn't ideal.

Reply:
Otoh, leaving the emoji in the text means that it too could be localized, if ⚠️ isn't as meaningful as a warning icon in another language.